app/src/main/java/com/framgia/forder/screen/managerdetail/ManagerDetailViewModel.java

```diff
@@ -25,4 +32,65 @@ public void onStop() {
     public void setPresenter(ManagerDetailContract.Presenter presenter) {
         mPresenter = presenter;
     }
+
+    public String getUserName() {
+        if (mUser != null) {
+            return mUser.getName();
+        }
+        return "";
+    }
+
+    public String getEmail() {
+        if (mUser != null) {
+            return mUser.getEmail();
+        }
+        return "";
+    }
+
+    public String getChatWorkId() {
+        if (mUser != null) {
+            return mUser.getChatworkId();
+        }
+        return "";
+    }
+
+    public String getJoinDate() {
+        if (mUser != null) {
+            return mUser.getFormatDate();
+        }
+        return "";
+    }
+
+    public String getDescription() {
+        if (mUser != null) {
+            return mUser.getDescription();
+        }
+        return "";
+    }
+
+    public String getRole() {
+        if (mUser != null) {
+            return mUser.getRole();
+        }
+        return "";
+    }
+
+    public String getAvatar() {
+        if (mUser.getAvatar() != null && mUser.getAvatar().getImage() != null) {
+            mUser.getAvatar().getImage().getUrl();
```

Comment:
ERROR: [STYLE][RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT] Return value of com.framgia.forder.data.model.Image.getUrl() ignored, but method has no side effect
http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT